client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-table/index.tsx

```diff
@@ -39,77 +45,132 @@ const SiteTable = ( { isLoading, columns, items }: Props, ref: Ref< HTMLTableEle
-													icon={ info }
-													onClick={ () => onShowLicenseInfo( column.key ) }
+													className="site-table__favorite-icon"
+													size={ 24 }
+													icon={ starFilled }
 												/>
 											) }
-										</span>
-									</SiteSort>
+											<span className={ classNames( index === 0 && 'site-table-site-title' ) }>
+												{ column.title }
+
+												{ column.showInfo && (
+													<Icon
+														className="site-table__tooltip-icon"
+														size={ 16 }
+														icon={ info }
+														onClick={ () => onShowLicenseInfo( column.key ) }
+													/>
+												) }
+											</span>
+										</SiteSort>
+									</th>
+								) ) }
+								<th colSpan={ 3 }>
+									<div className="plugin-common-table__bulk-actions">
+										<EditButton isLargeScreen sites={ items } isLoading={ isLoading } />
+									</div>
 								</th>
+							</>
+						) }
+					</tr>
+				</thead>
+				<tbody>
+					{ isLoading ? (
+						<tr>
+							{ columns.map( ( column ) => (
+								<td className="site-table__tr-loading" key={ column.key }>
+									<TextPlaceholder />
+								</td>
 							) ) }
-							<th colSpan={ 3 }>
-								<div className="plugin-common-table__bulk-actions">
-									<EditButton isLargeScreen sites={ items } isLoading={ isLoading } />
-								</div>
-							</th>
-						</>
-					) }
-				</tr>
-			</thead>
-			<tbody>
-				{ isLoading ? (
-					<tr>
-						{ columns.map( ( column ) => (
-							<td className="site-table__tr-loading" key={ column.key }>
+							<td>
 								<TextPlaceholder />
 							</td>
-						) ) }
-						<td>
-							<TextPlaceholder />
-						</td>
-					</tr>
-				) : (
-					items.map( ( item, index ) => {
-						const blogId = item.site.value.blog_id;
+						</tr>
+					) : (
+						items.map( ( item, index ) => {
+							const blogId = item.site.value.blog_id;
 
-						return (
-							<SiteTableRow
-								index={ index }
-								item={ item }
-								columns={ columns }
-								key={ `table-row-${ blogId }` }
-								isExpanded={ expandedRow === blogId }
-								setExpanded={ () => setExpanded( blogId ) }
-							/>
-						);
-					} )
-				) }
-			</tbody>
-		</table>
+							return (
+								<SiteTableRow
+									index={ index }
+									item={ item }
+									columns={ columns }
+									key={ `table-row-${ blogId }` }
+									isExpanded={ expandedRow === blogId }
+									setExpanded={ () => setExpanded( blogId ) }
+								/>
+							);
+						} )
+					) }
+				</tbody>
+			</table>
+			{ shouldRenderAddSiteTourStep2 && (
+				<GuidedTour
+					className="jetpack-cloud-site-dashboard-new-site-added__guided-tour"
+					preferenceName="jetpack-cloud-site-dashboard-add-new-site-tour-step-2"
+					tours={ [
+						{
+							target: 'tr.is-latest-jetpack-connected-site td:first-of-type',
+							popoverPosition: 'bottom right',
+							title: translate( '🎉 Your new site is here' ),
```

Comment:
I'm not sure if this is ok.

I suggest keeping the emoji separate from the text:

NOTE: we might have to update the type of title.

```suggestion
							title: (
							<>
								{ String.fromCodePoint( 0x1f389 ) /* Celebration emoji 🎉 */ }
								{ translate( 'Your new site is here' ) }
							</>
						),
```

Reply:
Yes I thought the same initially. Here's an earlier commit where I changed this: 560c2f57843bd8dfad248fa2a38f86649dd1b128

However after some discussion in another PR, I changed this back. Here's the conversation: https://github.com/Automattic/wp-calypso/pull/85510#discussion_r1432498005

Is there something else you're aware of that we need to consider with the emoji within the translate string? I'm absolutely up for updating documentation relating to this too as I couldn't find anything clearly outlining the best approach.

Reply:
Noting that splitting the emoji out of the main string is less ideal for translators, as it gives them less flexibility to remove/reposition the emoji, and forces the emoji position.

Reply:
I don't know if there is any documentation. The best place to confirm this would be the ` #i18n` channel in Slack is the best place to confirm this.

Reply:
I'll go ahead and check there, thanks! Is there a known reason to prefer the way you suggested, or is it just something that you've seen elsewhere?

Reply:
TBH, I prefer the way it is currently. It was easy to read. It's just that I'm not sure if that is the preferred way to do it by the translation team.

Reply:
Yeah, I saw the `String.fromCodePoint()` method was used elsewhere but I have a tendency to abandon status quo when it doesn't make sense. So let's hope we get a good answer! :^)

Reply:
Thanks all for following up here! There has been a response by the i18n team and the preference is as @tbradsha suggested - they prefer the emoji to be part of the translatable string. So we should be good to go on this. Slack ref: p1703229258258059/1703212440.075029-slack-C02AED43D (I'll make a separate note to see where I can document this more clearly).